docs/pages/includes/config-reference/auth-service.yaml

```diff
@@ -266,6 +266,15 @@ auth_service:
         # "14h30m", "1h" etc.
         default_session_ttl: 12h
 
+        stable_unix_user_config:
+            # If set to true, SSH instances will use the same UID for each given
+            # username when automatically creating users. Defaults to false.
+            #enabled: true
+
+            # The range of UIDs (including both ends) used for automatic UID assignment.
+            #first_uid: 90000
+            #last_uid: 95000
```

Comment:
Can we uncomment these but set `enabled: false`?

I find it reads awkwardly when everything under the top level `stable_unix_user_config` is commented out.